vrk-kpa/suomidigi
# SUOPA-X Asynchronous paragraphs migration  [to beta]

To test: 
- `make fresh`
- Clear the caches
- Go to https://suomidigi.docker.sh/admin/paragraph-translation-migration 
    1. Click on **Migrate entities**
       - Wait until the migration has completed
       - Clear the caches
    2. Click again on **Migrate entities**
       - Wait until the migration has completed. This fixes missing paragraphs from the first run
- Clear the caches
- Go to English front page https://suomidigi.docker.sh/en
   - Some of the content should still be in Finnish.
   - Edit the page and save it once. 
   - Reload the page and check if the nodes have automatically switched to the translated ones. 
   - Edit the page and remove the ones which are not translated.
   - Check that the front page has updated and switch to FI translation. Check that the page is still intact and none of the references have been removed.
- Go to f.e. https://suomidigi.docker.sh/node/166 and open the english translation in to a new tab
- Compare these two contents and check that the layout is the same (meaning, the paragraphs work)
- Edit the english translation and remove the one of the paragraphs. Check that the original version (fi) stays intact
